PIM/API/05_ManageProducts.md

```diff
@@ -0,0 +1,427 @@
+
+[comment]: <> (CHECK!)
+
+To manage your products via API, you need to provide the required key-value pairs in your requests. The following elements are essential: 
+
+**Attribute key**
+
+Depending on the data you want to specify, you have to add the corresponding attribute, in its form as attribute key, to your request. For detailed information on the attribute key structure, see [Attributes](./03_KeyConcepts.md#attributes). 
+
+For a complete list of the attributes included in the attribute set of the product you want to address, you can check the corresponding *PIM* attribute set under *PIM > Settings > Attribute sets*. Alternatively, you can find a list of all existing *PIM* attributes under *PIM > Settings > Attributes*. Alternatively, you can get a list of attributes via API, see [List the attributes in an attribute set](./04_RetrieveEntityData.md#list-the-attributes-in-an-attribute-set).
+
+Bear in mind that keys are customer-defined. Therefore, the fields displayed in the following request samples should just serve as an example. For a practical example on how to use the request samples, see [How to use the request samples](./02_GetStarted.md#how-to-use-the-request-samples). 
+
+> [Caution] In the latest version of the *PIM* and *DataHub* modules, it is possible to modify the attribute key. Bear in mind, though, that attribute keys are used for API access. Changing the attribute key, which is strongly discouraged, has an impact on the API naming for all affected entities where this attribute is included. That means that all external API consumers, workflows, webhooks, or other custom code need to be adjusted accordingly in case any of them address this attribute.
+
+**Entity ID**
+
+You may need to provide an entity ID value to a specific attribute key. Any ID values you need, for example, the **attributeSetId**, must be an existing value in the *DataHub* module. You can find out an entity ID via user interface or via API. For detailed information, see [Retrieve entity data](./04_RetrieveEntityData.md).
+
+**Data type**
+
+The value you can give a specific attribute depends on the attribute's data type. For detailed information on the data types, see [Data types](../../DataHub/UserInterface/04_DataTypeList.md).
+
+[comment]: <> (Zu prüfen: Attribute keys in kursiv oder nicht? Also *_pim_color* oder _pim_color, usw. In anderen Dateien es so gemacht, aber hier zu umständlich/verwirrend. Evtl. doch im Fließtext, damit es hervorgehoben ist, aber nicht in den Tabellen...)
+
+## The product object
+
+Products represent the goods and services you offer to your customers. They are described through a set of attributes. 
+
+The following table displays a list of all attributes contained in the *PIM basic set* (_pim_attribute_name) with their default data types and configuration. Additional fields, such as created and modified, are set by default in all entities and can be used to set filters. The required fields are marked in bold. 
+
```

Comment:
Tabelle unten: Ersetze e.g. mit for example siehe Style Guide https://actindo.atlassian.net/wiki/spaces/DOC/pages/96960517/Abbreviations